src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/AdditionalTerm.kt

```diff
@@ -0,0 +1,38 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity
+
+import org.hibernate.annotations.Fetch
+import org.hibernate.annotations.FetchMode
+import org.hibernate.annotations.LazyCollection
+import org.hibernate.annotations.LazyCollectionOption
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType.IDENTITY
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.OneToMany
+import javax.persistence.OrderBy
+import javax.persistence.Table
+import javax.validation.constraints.NotNull
+
+@Entity
+@Table(name = "additional_term")
+data class AdditionalTerm(
+  @Id
+  @GeneratedValue(strategy = IDENTITY)
+  @NotNull
+  val id: Long = -1,
+
+  @NotNull
+  val licenceId: Long = -1,
```

Comment:
This doesnt reference a licence object (i.e. table)?

Reply:
At present this is one way - Licence as the parent - which populates the lists of standard, additional and bespoke terms from the child tables on a findByXXX. I did have some "fun" getting the joins working so I suspect there might be a better way, but it is working as is. I'll read-up shortly.

Reply:
We can iterate on this :)